men_team_1/app/views/users/show.html.erb

```diff
@@ -7,4 +7,5 @@
 
 
 
-<%= link_to "Pairings Path", pairings_path(:id => @user.id) %>
\ No newline at end of file
+<%= link_to "Pairings Path", pairings_path(:id => @user.id)%></br>
+<%= link_to "#{@request.any? ? ('Request(' + @request.length.to_s + ')'):'Request'}", pairing_path %>
```

Comment:
This line is very difficult to parse for me. I would write it out in multiple steps to improve readability. One possibility:
```
<%
  requests_text = "Request"
  requests_text += " (#{@pending_mentees.length})" if @pending_mentees.any?
%>
<%= link_to requests_text, pairing_path %>
```